sycl/include/CL/sycl/context.hpp

```diff
@@ -216,6 +216,7 @@ class __SYCL_EXPORT context {
   ///
   /// \return a native handle, the type of which defined by the backend.
   template <backend BackendName>
+  __SYCL_DEPRECATED("Use SYCL-2020 sycl::get_native free function")
```

Comment:
This is hardly purpose of this patch, but should we start transitioning from using `class::get_native()` to `class::getNative()` (in general)?
https://github.com/intel/llvm/blob/c23fe4b94cbb95cfce3ee9ce9c0b6431fda1ec80/sycl/include/CL/sycl/backend.hpp#L67-L74

Reply:
Do you mean call to obj.getNative() at line 73? I'd prefer it to be done separately.
We'd need to make this function a friend, probably.
I wonder why current call doesn't issue the deprecated message. Any idea?

Reply:
> Do you mean call to obj.getNative() at line 73?

yes

> I'd prefer it to be done separately.

agree

> I wonder why current call doesn't issue the deprecated message. Any idea?

sycl.hpp is added as system header. Warnings from system headers are not reported.

Reply:
> sycl.hpp is added as system header. Warnings from system headers are not reported.

Good to know, thanks.